### DISCUSSION TITLE
./pocketbase serve command not found

### DISCUSSION THREAD

**Discussion by Visnicio** [pocketbase/pocketbase#2377] [Q&A] [answered]
I have a oracle cloud VPS running with CloudPanel installed to host my personal projects.

I have a project called guiadodev and that its supposed to be a blog.

I extracted the pocketbase linux file into the htdocs/guiadodev.vom/ folder but it seems to not recognize the command `./pocketbase serve`, even though i'm at the right folder

![image](https://user-images.githubusercontent.com/70033167/235036560-2c040ba5-4083-4ee4-9a39-c3663e0ff88b.png)

**Comment by ganigeorgiev** [comment, marked as answer]
I'm not sure what is the reason for the error. You could check with `ls -la` to ensure that the file is there and that it has exec permissions (_if not you can run something like `chmod u+x pocketbase`_).

Also make sure that you've downloaded the correct bundle for your VM. For Oracle Ampere A1 it should be the arm64 bundle, aka. https://github.com/pocketbase/pocketbase/releases/download/v0.15.2/pocketbase_0.15.2_linux_arm64.zip.

**Reply by Visnicio** [reply]
Its worked! Thanks mate

Now I just need to adress what's my domain port, 'cause none of 8443, 433 or 80 are working with `./pocketbase serve --https="guiadodev.com:80"`

**Reply by ganigeorgiev** [reply]
When starting with the `--http='guiadodev.com:80' --https='guiadodev.com:443'` we bind to the IP resolving behind the domain name, so make sure that:

- there is a valid dns record for the domain pointing to your vm
- the ip is properly resolving to your vm, aka. it is mapped in your `/etc/hosts` file (_you may find useful the comments from [issue#107](https://github.com/pocketbase/pocketbase/issues/107#issuecomment-1332372608)_)

**Reply by Visnicio** [reply]
I just didn't read the docs before, sorry.
The problem is that my server is supposed to host other projects, so I need a way to figure out how to put the pocketbase of this specific application on a reverse proxy.

I use CloudPanel as my server config software, so I have easy access to the virtual host file, just need to fix that now... I presume
